src/bun.js/bindings/bindings.cpp

```diff
@@ -4048,38 +4054,78 @@ class V8StackTraceIterator {
             return false;
         }
 
-        auto firstColon = line.find(':', openingParenthese + 1);
+        auto lineInner = StringView_slice(line, openingParenthese + 1, closingParenthese);
 
-        // if there is no colon, that means we only have a filename and no line number
+        {
+            auto marker1 = 0;
+            auto marker2 = lineInner.find(':', marker1);
 
-        //   at foo (native)
-        if (firstColon == WTF::notFound) {
-            frame.sourceURL = line.substring(openingParenthese + 1, closingParenthese - openingParenthese - 1);
-        } else {
-            // at foo (/path/to/file.js:
-            frame.sourceURL = line.substring(openingParenthese + 1, firstColon - openingParenthese - 1);
-        }
+            if (marker2 == WTF::notFound) {
```

Comment:
Can you add a comment somewhere (relevant) that explains why we are doing the search this way (making it more robust). This will help future us when we want to try to optimize smth and we can see oohh ya we do it this way for cross-platform goodness.